### PR TITLE
Changed methods to use profileViewModel instead of userViewModel in SignInScreen for testing

### PR DESCRIPTION
**Pull request by engjellismaili**

### CODE REVIEW

File: app/src/main/java/com/github/wanderwise_inc/app/ui/signin/LoginScreen.kt [hunk 1]
```diff
@@ -174,11 +177,11 @@ fun SignInButton(
                         val description = ""
                         val upVotes = 0
 
-                        val u = User(uid, properUsername, properEmail, properPhoneNumber, country, description, upVotes)
+                        val u = Profile(displayName = properUsername, userUid = uid, bio = description, profilePicture = user.photoUrl)
```
Comment (YofiY) [inline app/src/main/java/com/github/wanderwise_inc/app/ui/signin/LoginScreen.kt:180]: changing `u` to `newProfile` would improve readabliity

**Review by YofiY — CHANGES_REQUESTED** [review]
Good job! There are however some slight changes to be made in order to improve readability.

File: app/src/main/java/com/github/wanderwise_inc/app/ui/signin/LoginScreen.kt [hunk 2]
```diff
@@ -177,7 +177,7 @@ fun SignInButton(
                         val description = ""
                         val upVotes = 0
 
-                        val u = Profile(displayName = properUsername, userUid = uid, bio = description, profilePicture = user.photoUrl)
+                        val newProfile = Profile(displayName = properUsername, userUid = uid, bio = description, profilePicture = user.photoUrl)
```
Comment (ethangraham2001) [inline app/src/main/java/com/github/wanderwise_inc/app/ui/signin/LoginScreen.kt:180]: LGTM. More readable than before

File: app/src/main/java/com/github/wanderwise_inc/app/ui/signin/LoginScreen.kt [hunk 3]
```diff
@@ -192,8 +192,8 @@ fun SignInButton(
 
                                 // TODO Handle ERROR
                             }*/
-                            profileViewModel.setProfile(u)
-                            val queriedProfile = profileViewModel.getProfile(u.userUid).first()
+                            profileViewModel.setProfile(newProfile)
+                            val queriedProfile = profileViewModel.getProfile(newProfile.userUid).first()
```
Comment (ethangraham2001) [inline app/src/main/java/com/github/wanderwise_inc/app/ui/signin/LoginScreen.kt:196]: Looks consistent with API.

**Review by ethangraham2001 — APPROVED** [review]
Everything looks good. Looks like this will help everybody get up and running with writing their UI components

File: app/src/main/java/com/github/wanderwise_inc/app/ui/signin/LoginScreen.kt (hunk 1 above)
Comment (ethangraham2001) [reply app/src/main/java/com/github/wanderwise_inc/app/ui/signin/LoginScreen.kt]: The changes have been made. Resolving

**Review by YofiY — APPROVED** [review]
lgbt you can merge now.

**Pull request merged**